### PR TITLE
New register and email sending, tabs in usertable

### PR DESCRIPTION
**Pull request by vovikboyko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by AlexKaschuk — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/Auth/AuthService.cs [hunk 1]
```diff
@@ -218,31 +230,38 @@ public UserDTO GetUser(User user)
...
+            return (await _emailConfirmation.SendEmailAsync(
+                email: email,
+                subject: "Підтвердження реєстрації ",
+                message: $"Підтвердіть реєстрацію, перейшовши за :  <a href='{confirmationLink}'>посиланням</a> ",
+                title: "Адміністрація сайту EPlast"));
         }
 
         ///<inheritdoc/>
-        public async Task SendEmailRegistr(string confirmationLink, RegisterDto registerDto)
+        public async Task SendEmailReminder(string citiesUrl, UserDTO userDTO)
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Services/Auth/AuthService.cs:251]:
```suggestion
        public async Task SendEmailReminderAsync(string citiesUrl, UserDTO userDTO)
```

File: EPlast/EPlast.BLL/Services/Auth/AuthService.cs [hunk 2]
```diff
@@ -218,31 +230,38 @@ public UserDTO GetUser(User user)
...
-            await _userManager.UpdateAsync(user);  
-            await _emailConfirmation.SendEmailAsync(user.Email, "Підтвердження реєстрації ",
-                $"Підтвердіть реєстрацію, перейшовши за :  <a href='{confirmationLink}'>посиланням</a> ", "Адміністрація сайту EPlast");
+            await _emailConfirmation.SendEmailAsync(user.Email, "Вітаємо у системі!",
+                $"Ви успішно активували свій акаунт!\nНе забудьте доєднатись до осередку, перейшовши за :  <a href='{citiesUrl}'>посиланням</a> ", "Адміністрація сайту EPlast");
         }
 
         ///<inheritdoc/>
-        public async Task SendEmailReseting(string confirmationLink, ForgotPasswordDto forgotPasswordDto) 
+        public async Task SendEmailReseting(string confirmationLink, ForgotPasswordDto forgotPasswordDto)
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Services/Auth/AuthService.cs:260]:
```suggestion
        public async Task SendEmailResetingAsync(string confirmationLink, ForgotPasswordDto forgotPasswordDto)
```

**Review by voroniak — CHANGES_REQUESTED** [review]

**Comment by tetyanakutsko** [comment]
2code smells.. need to be fixed

**Review by MykhailoKlapchuk — APPROVED** [review]

File: EPlast/EPlast.BLL/DTO/UserProfiles/ShortUserInformationDTO.cs [hunk 3]
```diff
@@ -10,5 +10,7 @@ public class ShortUserInformationDTO
         public string LastName { get; set; }
         public DateTime? Birthday { get; set; }
         public GenderDTO Gender { get; set; }
+        public string Email { get; set; }
+        public bool EmailConfirmed { get; set; }
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/DTO/UserProfiles/ShortUserInformationDTO.cs:14]: Maybe, should rename this property to "IsEmailConfirmed"?

**Review by sooooolo — APPROVED** [review]

**Pull request closed without merging**